ar_exercise/exercise.rb

```diff
@@ -31,41 +34,91 @@ class Apartment < ActiveRecord::Base
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
+# Before running pry, run file through ruby (fil)
+
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+all_tenants = Tenant.first
+
 # get all tenants older than 65
+all_tenants = Tenant.where("age: > ?" "65")
+
 # get all apartments whose price is greater than $2300
+all_apartments = Apartment.where("age: > ?" "2300")
+
 # get the apartment with the address "6005 Damien Corners"
+all_apartments = Apartment.where(address:"6005 Damien Corners")
```

Comment:
although I wouldn't use `all_apartments` as a variable name, but something that indicates its one apartment, e.g.:

`current_apartment` or `apartment_at_damien_corners`.